### PR TITLE
Update vpc validation

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

File: src/api/app/views/webui/cloud/upload_jobs/new.html.haml [hunk 1]
```diff
@@ -22,7 +22,7 @@
     = upload_job_form.label 'cloud_backend_upload_job[region]', "Region:"
     = select_tag('cloud_backend_upload_job[region]', options_for_select(@ec2_regions))
   %p
-    = upload_job_form.label 'cloud_backend_upload_job[vpc_subnet_id]', 'VPC Subnet ID (optional):'
+    = upload_job_form.label 'cloud_backend_upload_job[vpc_subnet_id]', 'VPC Subnet ID (eg. subnet-12345678, optional):'
```
Comment (ChrisBr) [inline src/api/app/views/webui/cloud/upload_jobs/new.html.haml]: [nitpick] Maybe we can add another hint that we will create a temp VPC if the user does not provide one

Comment (bgeuken) [reply src/api/app/views/webui/cloud/upload_jobs/new.html.haml]: @ChrisBr I added a tooltip

File: src/api/app/views/webui/user/show.html.haml [hunk 2]
```diff
@@ -12,6 +12,10 @@
       %br/
     %span#home-username= @displayed_user.login if @displayed_user.login
   / mail
+  %p
+    = label_tag 'cloud_backend_upload_job[vpc_subnet_id]', 'VPC Subnet ID (eg. subnet-12345678, optional):'
```
Comment (ChrisBr) [inline src/api/app/views/webui/user/show.html.haml]: Huh? Why on the user show page? I guess this is a leftover from testing?

Comment (bgeuken) [reply src/api/app/views/webui/user/show.html.haml]: yes

**Pull request merged**